.github/workflows/performance-test.yml

```diff
@@ -25,6 +25,10 @@ jobs:
             iterations: 4
             jfr-delay: "50s"
 
+          - location: skanetrafiken
```

Comment:
All the other locations are named after the place, not the company, so can you rename it to `skane` please?

Reply:
Thats not the only data that is included. This dataset includes GTFS data for Denmark as well + some other trips outside Skåne. It would rather keep it as it is. We may have some other public transport providers from Denmark/Sweden using OTP.